akka-http-core/src/test/scala/akka/http/impl/engine/client/HighLevelOutgoingConnectionSpec.scala

```diff
@@ -38,8 +39,7 @@ class HighLevelOutgoingConnectionSpec extends AkkaSpec {
         .mapAsync(4)(_.entity.toStrict(1.second))
         .map { r ⇒ val s = r.data.utf8String; log.debug(s); s.toInt }
         .runFold(0)(_ + _)
-
-      result.futureValue(PatienceConfig(10.seconds)) shouldEqual N * (N + 1) / 2
+      result.futureValue(PatienceConfig(10.seconds), Position.here) should ===(N * (N + 1) / 2)
```

Comment:
There is another [`futureValue`](http://doc.scalatest.org/3.0.0/index.html#org.scalatest.concurrent.Futures$FutureConcept) method, that accepts `Timeout` explicit parameter. Then this would become:

```scala
result.futureValue(Timeout(10.seconds)) should ===(N * (N + 1) / 2)
```

It also needs additional import:

```scala
import org.scalatest.concurrent.PatienceConfiguration.Timeout
```